db/migrate/20231206192754_delete_all_duration_rates.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+class DeleteAllDurationRates < ActiveRecord::Migration[7.0]
+  class DurationRate < ApplicationRecord; end
+
+  def change
+    DurationRate.delete_all
```

Comment:
We should make a note to do some manual testing around this before releasing

Reply:
You mean like checking that price policies aren't broken? Any other cases you can think of?

Reply:
If we have any duration rate records on production, we should check if any orders have been placed against them.  I don't think anyone is using this yet but I just want to be safe